.github/workflows/issue-assignment.yml

```
@@ -0,0 +1,57 @@
+  issue_comment:
+    types: [created]
+
+permissions:
+  issues: write
+
+jobs:
+  respond-to-assign-request:
+    runs-on: ubuntu-latest
+    if: |
+      github.event.issue.pull_request == null &&
+      (
```

actionlint (1.7.10)

[error] undefined function "toLower". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)